.circleci/config.yml

```diff
@@ -63,7 +63,17 @@ macos_brew_update: &macos_brew_update
     brew install expect
     brew install libomp
 
-
+ios_brew_update: &ios_brew_update
+  name: Brew update and install iOS toolchains
+  no_output_timeout: "1h"
+  command: |
+    set -ex
+    brew update
+    brew unlink parallel
+    brew install moreutils
+    brew link parallel --overwrite
+    brew install expect
+    brew install libtool
```

Comment:
This seems like a direct copy paste `binary_macos_brew_update` (except that it is missing the `brew untap caskroom/homebrew-cask` command, but *arguably* it should do that here too). Can't we just use `binary_macos_brew_update` instead (perhaps renaming it to more clearly say something like, `macos_brew_update_nolibomp`)? Without deduplicating, there's now **three** places you'd have to update if there were tweaks to this block.

Reply:
I also added the `libtool` at the end. I remember  I ran into some error when running `brew untap caskroom/homebrew-cask`.